execution_result.go

```diff
@@ -0,0 +1,26 @@
+package flow
+
+import "github.com/onflow/flow-go-sdk/crypto"
+
+type ExecutionResult struct {
+	PreviousResultID Identifier // commit of the previous ER
+	BlockID          Identifier // commit of the current block
+	Chunks           []*Chunk
+	ServiceEvents    []*ServiceEvent
+}
+
+type Chunk struct {
+	CollectionIndex      uint
```

Comment:
Shouldn't this be defined as uint32?
```suggestion
	CollectionIndex      uint32
```

Reply:
In `flow-go` it is `uint`